pkg/rook/config.go

```diff
@@ -15,79 +15,38 @@
 package rook
 
 const (
-	ClusterIdDefaultValue                        = "rook-ceph"
-	MonitorConfigMapNameDefaultValue             = "rook-ceph-mon-endpoints"
-	MonitorConfigMapDataKeyDefaultValue          = "csi-cluster-config-json"
-	CSIRBDProvisionerSecretNameDefaultValue      = "rook-csi-rbd-provisioner"
-	CSIRBDNodeSecretNameDefaultValue             = "rook-csi-rbd-node"
-	StorageClassAllowVolumeExpansionDefaultValue = true
-	StorageClassFSTypeDefaultValue               = "ext4"
-	StorageClassImageFeaturesDefaultValue        = "layering,exclusive-lock,object-map,fast-diff"
-	StorageClassReclaimPolicyDefaultValue        = "Delete"
-	StorageClassVolumeBindingModeDefaultValue    = "Immediate"
-	CSIDriverNameDefaultValue                    = "rook-ceph.rbd.csi.ceph.com"
-	NamespaceDefaultValue                        = "rook-ceph"
-	EnableRBDStatsDefaultValue                   = false
-	BurstFactorDefaultValue                      = 10
-	BurstDurationInSecondsDefaultValue           = 15
-
-	BucketProvisionerDefaultValue = "rook-ceph.ceph.rook.io/bucket"
+	ClusterIdDefaultValue                 = "rook-ceph"
```

Comment:
Is there a reason why we will have the `rook` constants here?

Reply:
This stuff is needed for the `Bucket` reconciler and will be removed in the next refactoring.